src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/set/autocomplete.ts

```diff
@@ -39,3 +98,45 @@ export async function autocomplete(
 
   return [];
 }
+
+const getProjectRoutingCommonCompletionItems = (): ISuggestionItem[] => {
+  return [
+    {
+      label: '_alias:_origin',
+      text: '_alias:_origin',
+      kind: 'Value',
+      detail: i18n.translate(
+        'kbn-esql-ast.esql.autocomplete.set.projectRouting.currentProjectDoc',
+        {
+          defaultMessage: 'Search only the current project',
+        }
+      ),
+      sortText: '1',
+    },
+    {
+      label: '_alias: *',
+      text: '_alias: *',
+      kind: 'Value',
+      detail: i18n.translate('kbn-esql-ast.esql.autocomplete.set.projectRouting.allProjectsDoc', {
+        defaultMessage: 'Search all projects',
+      }),
+      sortText: '1',
+    },
+    {
+      label: '_alias:* AND NOT _alias:_origin',
```

Comment:
Sebastian let's remove this suggestion as they are not going to suggest AND expressions on the tech preview (I also removed it from the examples)